common-api/README.md

```diff
@@ -0,0 +1,10 @@
+# Common API
+
+Common utilities, services, data models between the Workbench and Public Data Browser.
+
+## Data model distinction
+
+Some models, e.g. cdr/model/Concept.java, may be used with either a registered CDR
+database or a public CDR database (depending on which app it is being used in). For the
+public DB, the data may have gone through additional deidentification, e.g. for Concept the
```

Comment:
"may have gone through"  should be "has gone through" .. :) There is no may have.

Reply:
I meant this on a table-to-table basis, e.g. the concept relationship table wouldn't have gone through any deidentification. Tweaked wording.